python/ray/serve/_private/router.py

```diff
@@ -390,20 +631,28 @@ def __init__(
         controller_handle: ActorHandle,
         deployment_name: str,
         event_loop: asyncio.BaseEventLoop = None,
-        _stream: bool = False,
+        _use_new_routing: bool = False,
     ):
         """Router process incoming queries: assign a replica.
```

Comment:
This docstring is a bit awkwardly phrased; could you rephrase it?

Reply:
on it